pkg/k8sclient/core_test.go

```diff
@@ -276,23 +295,22 @@ func (suite *CoreTestSuite) TestCreateNamespace() {
 		{
 			name: "create empty",
 			fields: fields{
-				ClientSet:   suite.kubeClient.ClientSet,
+				ClientSet:   fake.NewSimpleClientset(), // use fresh client
```

Comment:
The comment does not add any new info and should be removed.

Reply:
Fixed